docs/Guides/Ecosystem.md

```diff
@@ -155,6 +155,10 @@ section.
 - [`@ethicdevs/fastify-git-server`](https://github.com/EthicDevs/fastify-git-server)
   A plugin to easily create git server and make one/many Git repositories available
   for clone/fetch/push through the standard `git` (over http) commands.
+- [`@fastify-userland/request-id`](https://github.com/fastify-userland/request-id)
+  Fastify Request ID Plugin
+- [`@fastify-userland/typeorm-query-runner`](https://github.com/fastify-userland/typeorm-query-runner)
+  Fastify typeorm QueryRunner plugin
```

Comment:
I must protest the name `@fastify-userland`. It implies the Fastify team is maintaining some set of external modules.

I suggest we revert this PR until this issue can be resolved.

attn: @fastify/core

Reply:
> I must protest the name @fastify-userland. It implies the Fastify team is maintaining some set of external modules.

I can delete this org if necessary.

> I suggest we revert this PR until this issue can be resolved.

OK. I agree